models/area.go

```diff
@@ -95,12 +92,16 @@ func (a *AreaParams) SetAreaType(ctx context.Context) {
 type AreasDataResults struct {
 	Code          string           `json:"code"`
 	Name          *string          `json:"name"`
-	GeometricData [][][2]float64   `json:"geometry"`
+	GeometricData GeoData          `json:"geometry"`
```

Comment:
This PR is changing the structure of the AreasDataResults struct, but the PR description makes no mention of this change. Is this intentional?

Reply:
Yes this was intentional, I had to do this because with the current structure it was never worked or did pass the unmarshaller if we had checked for errors. 

```
func initialiseStubbedAreaData(_ context.Context) (map[string]models.AreasDataResults, error) {
	geoData := make(map[string]models.AreasDataResults, 2)
	for _, geoDataFile := range fls {
		var data models.AreasDataResults
		if err := json.Unmarshal([]byte(geoDataFile), &data); err != nil {
			return nil, err
		}
		geoData[data.Code] = data
	}
	return geoData, nil
}
```
This is because here the range ```fls``` seems to be slice of string constants and if you look at one constant say the following

`E92000001PropertyData = `{"code": "E92000001","date_end": null,"date_start": "Thu, 01 Jan 2009 00:00:00 GMT","name": "England","name_welsh": "Lloegr", "geometry": {"coordinates": [[[[-2.02938571313103, 55.76887759555741],[-2.028398524545316, 55.76825328366719],[-2.025372831650281, 55.76935547784223],[-2.021543541644557, 55.77061756272807],[-2.017688237957491, 55.77146457301031]]]],"type": "MultiPolygon"},"visible": true}``

This never worked as this has the coordinates attributes and it was again  ```[][][][2]float64``` rather than ```[][][2]float64```. So i had to change these to make the code pass. now if the coordinates attribute  is needed or not , I am not sure and probably Nathan should make a call on this. Hope this helps.